project/README.md

```diff
@@ -0,0 +1,55 @@
+Getting Started with ScuttleMesh
+================================
+
+**ScuttleMesh** is [Secure Scuttlebutt](https://github.com/ssbc/secure-scuttlebutt) over a self-configuring encrypted mesh network formed by Raspberry Pis.
+
```

Comment:
I common pattern I use is to give a quick one-liner on the main tech used, to help orient people minimally without sending them away into linked websites.

What about these as options?
- [x] **ScuttleMesh**Secure Scuttlebutt over a self-configuring encrypted mesh network formed by Raspberry Pis.
- [ ] **Secure Scuttlebutt (SSB).** A database of unforgeable append-only feeds, optimized for efficient replication for peer to peer protocols.
- [ ] **Mesh Orange.** A small ramdisk system running modern Debian to support mesh networking on ARM single-board computers.